spec/var-op-bash.test.sh

```diff
@@ -328,7 +329,8 @@ A - A
 status=0
 ## END
 ## OK osh STDOUT:
-status=1
+- y
```

Comment:
One trick we could use to make bash and OSH match is to do

```
sq=\'
s/y/${sq}y${sq}/
```

or vice versa

And then add a comment that this "papers over" an irrelevant or acceptable difference

---

I am not saying we need to do this, only if you think it makes the tests more understandable

I think if the only difference is quoting, then we should feel free to "adjust" the output, as long as it doesn't "hide" any other bugs!

(We can also use strings like  `y55 y56 y57` etc. to reduce the changes of accidentally hiding a bug, i.e. add a random number)

Reply:
Does 0653c5e29 work for you? To avoid a false positive, I match the entire line for replacement.

Reply:
yes looks good thanks!